src/features/releases/hooks/delete-checks.ts

```diff
@@ -0,0 +1,31 @@
+import { sbvrUtils, hooks, permissions } from '@balena/pinejs';
```

Comment:
This hook should really be in a feature that would create the `should_be_managed_by__release` relationship, presumably something supervisor related

Reply:
Need some guidance about what this means (in terms of a practical resolution) ...fyi ... @xginn8 (thanks)

Reply:
@xginn8 just some clarification... so instead of PRERUNning when DELETing a `release`, this code will now PRERUN when DELETing a `device`?  :point_left: proceeding with that understanding til I hear otherwise.